src/components/maps/MapSelector.tsx

```diff
@@ -1,4 +1,37 @@
-export const MAP_STYLES = {
-  outdoor: 'https://api.maptiler.com/maps/outdoor-v2/style.json?key=ejjLkz58mUNz9TgNs0Ed',
-  dataviz: 'https://api.maptiler.com/maps/dataviz/style.json?key=ejjLkz58mUNz9TgNs0Ed'
+const MAPTILER_KEY = process.env.NEXT_PUBLIC_MAPTILER_API_KEY !== undefined ? process.env.NEXT_PUBLIC_MAPTILER_API_KEY : 'key'
+export const MAP_STYLES: MapStyles = {
+  outdoor: {
+    style: `https://api.maptiler.com/maps/outdoor-v2/style.json?key=${MAPTILER_KEY}`,
+    imgUrl: 'https://docs.maptiler.com/sdk-js/api/map-styles/img/style-outdoor-v2.jpeg'
+  },
+  minimal: {
+    style: `https://api.maptiler.com/maps/dataviz/style.json?key=${MAPTILER_KEY}`,
+    imgUrl: 'https://docs.maptiler.com/sdk-js/api/map-styles/img/style-bright-v2-pastel.jpeg'
```

Comment:
Should that be 
```
'https://docs.maptiler.com/sdk-js/api/map-styles/img/style-dataviz.jpeg'
```
?

The other styles have matching urls in `style` and `imgUrl`.

Reply:
no, this was actually on purpose, because our dataviz style did not quite match the one in the picture, that's why i picked the one that looked most similar:
![image](https://github.com/OpenBeta/open-tacos/assets/51745385/88efcaad-1175-4ef9-a8f3-86d4fe2951c6)
if you wanna have a look:
https://docs.maptiler.com/sdk-js/api/map-styles/#referencemapstyle

this is where i got the imageurls from

Reply:
Cool, I figured it was probably something like that.